Project/Assets/ML-Agents/Examples/SharedAssets/Prefabs/Targets/Target.prefab.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
removed.

Reply:
Can you do a pass on the currently existing prefab? Maybe there are some more unused prefabs that we can remove in a next PR.

Reply:
there may be some prefabs that are used in my current Updated Worm PR. after both are merged i can do an additional PR to remove unused prefabs. *I need to remove/consolidate some from walker as well ( instead of 4 diff walker prefabs i want to do initialization setup similar to how it's used in this PR)